### PR TITLE
correct linting issues from ESLint

### PR DESCRIPTION
**Pull request by expe**
DO NOT MERGE until OBT is final.

Changes based on new lint rules in OBT / ESLint.

### CODE REVIEW

File: test/foo.test.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 /*global describe, it*/
-'use strict';
 
-var expect = require('expect.js');
+const expect = require('expect.js');
 
-var Foo = require('./../src/js/foo');
+const foo = require('./../src/js/foo');
```
Comment (AlbertoElias) [inline test/foo.test.js:5]: why is that not a capital F?

Comment (expe) [inline test/foo.test.js:5]: Capitals should be reserved for constructors. In this example it is just a function? It's not doing anything constructor-like.

Comment (AlbertoElias) [inline test/foo.test.js:5]: It's just a component blueprint, but we should be thinking that if it's a module, for it to act like a constructor. Feel free to improve the blueprint to show that behaviour

**Comment by onishiweb** [comment]
Deprecating this module in #9 closing this now.

**Pull request closed without merging**